Sharpmake/Target.cs

```diff
@@ -7,6 +7,8 @@
 using System.Diagnostics;
 using System.Linq;
 using System.Reflection;
+using static Sharpmake.EngineTarget;
```

Comment:
this do not belong in sharpmake codebase.
Define your own Target and fragments in your sharpmake scripts.